app/routes/traces/patch.js

```diff
@@ -0,0 +1,61 @@
+import Boom from '@hapi/boom';
+import Joi from '@hapi/joi';
+import db from '../../services/db';
+import logger from '../../services/logger';
+import traces from '../../models/traces';
+import config from 'config';
+
+const idLength = config.get('idLength');
+
```

Comment:
Could you add some documentation to make clear that `description` is the only accepted parameter / thing one can `patch`?